widget/storybook/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@rango-dev/storybook",
+  "version": "0.31.0",
+  "license": "MIT",
+  "type": "module",
+  "main": "./storybook-static/index.html",
+  "private": true,
+  "files": [
+    "storybook-static",
+    "src"
+  ],
+  "scripts": {
+    "dev": "storybook dev -p 3000",
+    "build": "storybook build && cp ./vercel.json ./storybook-static",
+    "ts-check": "tsc --declaration --emitDeclarationOnly -p ./tsconfig.json",
+    "type-checking": "tsc --declaration --emitDeclarationOnly",
+    "clean": "rimraf dist",
+    "format": "prettier --write '{.,src}/**/*.{ts,tsx}'",
+    "lint": "eslint \"**/*.{ts,tsx}\" --ignore-path ../../.eslintignore"
+  },
+  "peerDependencies": {
+    "react": ">=16"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.2",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.4",
+    "@chromatic-com/storybook": "^1",
+    "@storybook/addon-essentials": "^8.0.8",
+    "@storybook/addon-info": "^5.3.21",
+    "@storybook/addon-links": "^8.0.8",
+    "@storybook/addons": "^7.1.1",
+    "@storybook/react": "^8.0.8",
+    "@storybook/react-vite": "^8.0.8",
+    "@storybook/react-webpack5": "^8.0.8",
+    "@types/react": "^18.0.25",
+    "@types/react-dom": "^18.0.8",
+    "babel-loader": "^9.1.0",
+    "husky": "^8.0.2",
+    "react": "^18.2.0",
+    "react-is": "^18.2.0",
+    "storybook": "^8.0.8"
+  },
+  "dependencies": {
```

Comment:
Seems dependencies are copy pasted from `ui`, please make sure keeping those that neede. (same applies to devDependencies)

Reply:
You missed to remove `husky` (you can remove from `/ui` as well, it should be there as well).
And also `dependency` and `devDependencies` is not correct. please change it to something like this:
```json
  "devDependencies": {
    "@types/react": "^18.0.25",
    "@types/react-dom": "^18.0.8",
  },
  "dependencies": {
     "@chromatic-com/storybook": "^1",
    "@storybook/addon-essentials": "^8.0.8",
    "@storybook/addon-info": "^5.3.21",
    "@storybook/addon-links": "^8.0.8",
    "@storybook/addons": "^7.1.1",
    "@storybook/react": "^8.0.8",
    "@storybook/react-vite": "^8.0.8",
    "@storybook/react-webpack5": "^8.0.8",
    "@rango-dev/ui": "^0.31.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
     "react-is": "^18.2.0",
    "storybook": "^8.0.8"
  },
```

Reply:
I also not sure why `@chromatic-com/storybook` has been added. Do we need that?